packages/build/src/install/functions.js

```diff
@@ -2,18 +2,18 @@ const { dirname } = require('path')
 
 const readdirp = require('readdirp')
 
-const { logInstallFunctionDependencies } = require('../log/messages/install')
+const { logInstallFunctionDeps } = require('../log/messages/install')
```

Comment:
[sand] I prefer the non abbreviated `logInstallFunctionDependencies` name

Reply:
:+1: